pkg/cluster/service.go

```diff
@@ -3,26 +3,28 @@ package cluster
 import (
 	"context"
 	"encoding/base64"
+	"fmt"
 	"reflect"
 	"time"
 
 	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
 	"github.com/codeready-toolchain/toolchain-common/pkg/apis"
-	"k8s.io/client-go/rest"
-
 	"github.com/go-logr/logr"
 	"github.com/pkg/errors"
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 const (
-	labelType             = "type"
 	labelNamespace        = "namespace"
 	labelOwnerClusterName = "ownerClusterName"
+	LabelType             = "type"
+	// labelClusterRolePrefix is the prefix that defines the cluster role as label key
+	labelClusterRolePrefix = "cluster-role.toolchain.dev.openshift.com"
```

Comment:
we could replace the `toolchain.dev.openshift.com` suffix with [LabelKeyPrefix](https://github.com/codeready-toolchain/api/blob/master/api/v1alpha1/labels.go#L5)

Reply:
Great, I missed that! I'll change it! Thanks!

Reply:
changed in [5fd587a46e83cad1](https://github.com/mfrancisc/toolchain-common/commit/5fd587a46e83cad164b50f71c52895b9fce79610)

Reply:
did we finally decide to drop the `.toolchain.dev.openshift.com` part in the label key?

Reply:
no, I've not dropped it, since being appended in `RoleLabel` function to the `cluster-role` prefix

Reply:
my suggestion was to have something like this:
```
labelClusterRolePrefix = "cluster-role." + toolchainv1alpha1.LabelKeyPrefix
```
otherwise, you have to remember appending `toolchainv1alpha1.LabelKeyPrefix` to `labelClusterRolePrefix` everywhere it's used :/ (hence, it could be error-prone)